### PR TITLE
fix: Fix responsive for workspaces and workspace page

### PR DESCRIPTION
**Pull request by BrunoQuaresma**
Related to #3185 

https://user-images.githubusercontent.com/3165839/180859060-3b9e72bd-1b62-4edb-a49c-ed483deead6d.mov

Screen Shot 2022-07-25 at 16 27 28
Screen Shot 2022-07-25 at 16 25 12

### CODE REVIEW

File: site/src/components/BuildsTable/BuildsTable.tsx [hunk 1]
```diff
@@ -37,72 +38,74 @@ export const BuildsTable: FC<BuildsTableProps> = ({ builds, className }) => {
...
-                    <KeyboardArrowRight className={styles.arrowRight} />
-                  </div>
-                </TableCellLink>
-              </TableRow>
-            )
-          })}
-
-        {builds && builds.length === 0 && (
+    <TableContainer className={className}>
+      <Table data-testid="builds-table">
```
Comment (Kira-Pilot) [inline site/src/components/BuildsTable/BuildsTable.tsx:42]: Do we need a test id here or can we `findByRole` using `table`?

**Comment by Kira-Pilot** [comment]
I'm seeing a couple of button glitches in the snapshots. [Example](https://www.chromatic.com/test?appId=624de63c6aacee003aa84340&id=62def165432045a2e602d03a)

File: site/src/components/Table/Table.test.tsx [hunk 2]
```diff
@@ -1,81 +0,0 @@
-import { screen } from "@testing-library/react"
-import { render } from "../../testHelpers/renderHelpers"
-import { Column, Table } from "./Table"
-
-interface TestData {
```
Comment (Kira-Pilot) [inline site/src/components/Table/Table.test.tsx:5]: Why did we delete the test file?

**Comment by Kira-Pilot** [comment]
Just a couple of questions. What an improvement! 🥳

File: site/src/components/BuildsTable/BuildsTable.tsx (hunk 1 above)
Comment (BrunoQuaresma) [reply site/src/components/BuildsTable/BuildsTable.tsx:42]: I didn't know table is a role, lets try that.

Comment (BrunoQuaresma) [reply site/src/components/BuildsTable/BuildsTable.tsx:42]: Ahh, but there are two tables on the page, the resources table and the builds-table.

**Comment by BrunoQuaresma** [comment]
> I'm seeing a couple of button glitches in the snapshots. [Example](https://www.chromatic.com/test?appId=624de63c6aacee003aa84340&id=62def165432045a2e602d03a)

Fixed, thanks for catching those.

File: site/src/components/Table/Table.test.tsx (hunk 2 above)
Comment (BrunoQuaresma) [reply site/src/components/Table/Table.test.tsx:5]: We are not using those components anymore. They were wrappers around the material UI ones that we used in V1 that are no longer necessary.

File: site/src/components/BuildsTable/BuildsTable.tsx (hunk 1 above)
Comment (Kira-Pilot) [reply site/src/components/BuildsTable/BuildsTable.tsx:42]: I don't mind if you want to leave it as-is, but for future reference, you can use an `aria-describedby` attribute on it, and then do something like `getByRole('table', {description: 'builds table'})`

**Review by Kira-Pilot — APPROVED** [review]

File: site/src/components/PageHeader/PageHeader.tsx [hunk 3]
```diff
@@ -60,5 +65,11 @@ const useStyles = makeStyles((theme) => ({
 
   actions: {
     marginLeft: "auto",
+
+    [theme.breakpoints.down("md")]: {
+      marginTop: theme.spacing(3),
```
Comment (AbhineetJain) [inline site/src/components/PageHeader/PageHeader.tsx:70]: This extra margin is now pushing our primary content down when `actions` are not defined (this is visible in the updated stories). Maybe we should conditionally render the `Stack` containing the `actions`.

**Review by AbhineetJain — APPROVED** [review]
I have one minor comment, but I absolutely love this change! ❤️

File: site/src/components/PageHeader/PageHeader.tsx (hunk 3 above)
Comment (BrunoQuaresma) [reply site/src/components/PageHeader/PageHeader.tsx:70]: Good catch!

File: site/src/components/BuildsTable/BuildsTable.tsx (hunk 1 above)
Comment (BrunoQuaresma) [reply site/src/components/BuildsTable/BuildsTable.tsx:42]: Awesome! I'm doing it.

**Pull request merged**